apps/fluent-tester/src/E2E/common/BasePage.ts

```diff
@@ -77,21 +77,9 @@ export class BasePage {
       'Could not scroll to the ' + this._pageName + "'s Button. Please see Pipeline artifacts for more debugging information.";
 
     switch (platform) {
-      case Platform.Win32: {
-        const scrollDownKeys = [Keys.PAGE_DOWN];
-        await browser.waitUntil(
-          async () => {
-            await (await this._firstTestPageButton).addValue(scrollDownKeys);
-            scrollDownKeys.push(Keys.PAGE_DOWN);
-            return await (await this._pageButton).isDisplayed();
-          },
-          {
-            timeout: this.waitForUiEvent,
-            timeoutMsg: errorMsg,
-          },
-        );
+      case Platform.Win32:
+        // Not needed for Win32. Automatic scroll on click.
```

Comment:
nit: I would make the comments for win32 and macOS the same (minus the platform difference) for consistency since they seem to follow the same behavior